packages/gcf-utils/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "gcf-utils",
-  "version": "3.0.1",
+  "version": "3.0.3",
```

Comment:
That's interesting.  Are we manually bumping versions in this package?

Reply:
yes, release-please doesn't know how to deal with this package structure. I think the best solution would be to extract `gcf-utils` form the mono-repo.